lib/src/codepage.dart

```diff
@@ -5,158 +5,105 @@
 import 'dart:convert';
 import 'dart:typed_data';
 
+import 'codepages/unicode_iso8859.g.dart';
+
 /// The ISO-8859-2/Latin-2 (Eastern European) code page.
-final CodePage latin2 =
-    CodePage._bmp('latin-2', '$_ascii$_noControls$_top8859_2');
+///
+/// This is the authoritative mapping between ISO-8859-2 and
+/// Unicode text, as [specified by the Unicode Consortium](
+/// https://unicode.org/Public/MAPPINGS/ISO8859/8859-2.TXT).
```

Comment:
Bonus points for putting the table version here. You can get that from the mapping file:
`#	Table version:    2.0`

Reply:
Oooh, nice.